docs/GETTING-STARTED.md

```diff
@@ -0,0 +1,424 @@
+
+```
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+
+echo 'export MINIKUBE_WANTUPDATENOTIFICATION=false' >> ~/.bashrc
+echo 'export MINIKUBE_WANTREPORTERRORPROMPT=false' >> ~/.bashrc
+echo 'export MINIKUBE_HOME=$HOME' >> ~/.bashrc
+echo 'export CHANGE_MINIKUBE_NONE_USER=true' >> ~/.bashrc
+mkdir -p $HOME/.kube
+touch $HOME/.kube/config
+echo 'export KUBECONFIG=$HOME/.kube/config' >> ~/.bashrc
+source ~/.bashrc
+```
+
+Now, add yourself or the user running kubectl to a libvirt group so you can run the libvirt commands without sudo.
+
+```
+sudo usermod -a -G libvirt $(whoami)
+newgrp libvirt
+```
+
+#### Install kvm2 driver <a name="kvm2"></a>
+
+```
+curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2 && sudo mv docker-machine-driver-kvm2 /usr/local/bin/
+```
+     
+To check your GID membership for libvirt and docker
+
+```
+id $(whoami)
+```
+
+Start minikube
+
+After step above, minikube which was downloaded previously should be in /usr/local/bin.
+
+```
+minikube start --vm-driver kvm2
+```
+
+You should see:
+
+```
+    Starting local Kubernetes v1.10.0 cluster...
+    Starting VM...
+        Minikube ISO
+          153.08 MB / 153.08 MB [============================================] 100.00% 0s
+
+    Getting VM IP address...
+    Moving files into cluster...
+    Downloading kubeadm v1.10.0
+    Downloading kubelet v1.10.0
+    Finished Downloading kubelet v1.10.0
+    Finished Downloading kubeadm v1.10.0
+    Setting up certs...
+    Connecting to cluster...
+    Setting up kubeconfig...
+    Starting cluster components...
+    Kubectl is now configured to use the cluster.
+    Loading cached images from config file.
+```
+
+At any time you can verify that the minikube cluster is running by checking for the VM.
+
+```
+sudo virsh list --all
+```
+
+You should see:
+
+```
+    Id    Name                           State
+    ----------------------------------------------------
+    1     minikube                       running
+```
+
+### Install protobuf <a name="proto"></a>
+
+Install Google protobuf packages.
+
+```
+sudo dnf install -y protobuf dep
+sudo dnf install -y protoc-gen-go
+sudo dnf install -y protobuf-compiler
+```
+
+## Install and test network service mesh <a name="NSM"></a>
+
+### Download Network Service Mesh Code
+
+if GOPATH is set as environment variable
+
+```
+cd $GOPATH/src
+go get github.com/ligato/networkservicemesh
+```
+
+### Now finally, Network Service Mesh!
+
+Use make from the top level to generate the code, build docker images and install into minikube.
+
+```
+cd $GOPATH/src/github.com/ligato/networkservicemesh
+```
+
+#### Verify the code <a name="NSMverify"></a>
+
+To verify the code
+
+```
+make verify
+
+```
+
+#### Build the docker images
+
+```
+make docker-build
+```
+
+### Run Network Service Mesh
```

Comment:
Running `make` does not run NSM. Instead, it performs the three following tasks:

check
verify
docker-build

We can still merge this patch, but first this patch should either describe the steps for running NSM in Kubernetes or remove references to running NSM.